firstvoices/backend/views/search_all_entries_views.py

```diff
@@ -410,6 +412,54 @@
                     ),
                 ],
             ),
+            OpenApiParameter(
+                name="minWords",
+                description="Filter dictionary entries on the minimum number of words in their title."
+                " Only non-negative integer values are allowed.",
+                required=False,
+                default="",
+                type=int,
+                examples=[
+                    OpenApiExample(
+                        "2",
+                        value="2",
+                        description="Only return dictionary entries which have at least 2 words in their title.",
+                    ),
+                    OpenApiExample(
```

Comment:
Please don't include invalid examples in the docs. These are only meant to be examples of how people should use the API